### PR TITLE
Feature/course 1

### PR DESCRIPTION
**Pull request by SGInim**
PR (Pull Request) #1

According to what we have discussed with Ernesto, this Pull Request includes task1 and git

### CODE REVIEW

File: staff/sergi-pallice/course-1/Task1/contact.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <link rel="stylesheet" href="styles.css">
+<title>NYSL</title>
+</head>
+<body>
+  <div class="index-frame1">
+    <img src="Images/norcal_logo.png" class="logo"/>
+    <h1>NorCal Time Trial Series</h1>
+    <h3><a href="index.html">Home</a> | <a href="about.html">About </a>| <a href="contact.html">Contact</a></h3>
+  </div>
+<p>Please email us at info@nctts.org</p>
```
Comment (elolivier) [inline staff/sergi-pallice/course-1/Task1/contact.html]: This should go inside a div and image 3 is missing. Also email should be a link to the email.

File: staff/sergi-pallice/course-1/Task1/about.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+<title>NYSL</title>
+</head>
+<body>
+  <div class="index-frame1">
+    <img src="Images/norcal_logo.png" class="logo"/>
+    <h1>NorCal Time Trial Series</h1>
+    <h3><a href="index.html">Home</a> | <a href="about.html">About </a>| <a href="contact.html">Contact</a></h3>
+  </div>
+<div class="about-frame2">
+    <img src="Images/Image-2.png" class="soccer1"/>
```
Comment (elolivier) [inline staff/sergi-pallice/course-1/Task1/about.html:15]: Use same class for images in all pages in order to display them equal, class soccer1 doesn't exist on styles.css

File: staff/sergi-pallice/course-1/Task1/index.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+</head>
+<body>
+<div class="index-frame1">
+  <img src="Images/norcal_logo.png" class="logo"/>
+  <h1>NorCal Time Trial Series</h1>
+  <h3><a href="index.html">Home</a> | <a href="about.html">About </a>| <a href="contact.html">Contact</a></h3>
+</div>
+<div class="index-frame2">
+  <h2>Upcoming Events</h2>
+  <img src="Images/Image-1.png" class="logo"/>
```
Comment (elolivier) [inline staff/sergi-pallice/course-1/Task1/index.html]: Same as about image, if you are going to use class logo, use it on every image.

File: staff/sergi-pallice/course-1/Task1/styles.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+.index-frame1 img {
+  float: left;
+  margin-right: 20px;
+}
+
+.index-frame1 {
+  height: 140px;
+}
+
+.index-frame2 img {
```
Comment (elolivier) [inline staff/sergi-pallice/course-1/Task1/styles.css:25]: Same code that index-frame1 img, why two classes?

**Review by elolivier — CHANGES_REQUESTED** [review]

**Comment by elolivier** [comment]
Please review my comments

File: staff/sergi-pallice/course-1/Task1/about.html (hunk 2 above)
Comment (SGInim) [reply staff/sergi-pallice/course-1/Task1/about.html:15]: Fixed

**Review by elolivier — APPROVED** [review]

**Pull request merged**